code/modules/projectiles/projectile/bullets/pistol.dm

```diff
@@ -15,6 +15,14 @@
 	jostle_pain_mult = 6
 	rip_time = 1 SECONDS
 
+//Bubberstation addition
+/obj/projectile/bullet/c9mm/security
+	name = "9mm Security bullet"
+	damage = 20
+	embed_type = /datum/embedding/bullet/c9mm
+	weak_against_armour = TRUE
```

Comment:
honestly the security bullet changes should be pulled from the murphy PR, not added manually